app/controllers/api/v2/roles_controller.rb

```diff
@@ -39,7 +41,10 @@ def create
       param_group :role
 
       def update
-        process_response @role.update_attributes(role_params)
+        taxonomies = taxonomy_params!
+        result = @role.update_attributes(role_params)
+        @role.set_taxonomies(taxonomies)
```

Comment:
Not sure I understand the question. You mean how the authorization of taxonomies being assigned works?